README.md

```diff
@@ -62,13 +62,10 @@ nwbfile.external_resources.add_ref(
 path = 'test.nwb'
 with NWBHDF5IO(path, mode='w') as io:
     io.write(nwbfile)
-
-with NWBHDF5IO(path, mode='r', load_namespaces=True) as io:
-    read_nwbfile = io.read()
-    read_container = nwbfile.acquisition['test_ts']
-    read_table = nwbfile.acquisition['test_table']
-    print(nwbfile.external_resources.get_object_resources(read_container, 'unit'))
-    # TODO expand this example after paths for objects/keys is improved
+io = NWBHDF5IO(path, mode='r', load_namespaces=True)
```

Comment:
I would prefer to keep the context manager `with` style here, even for reads, because not using it can cause issues with open file handles on Windows machines.

Reply:
What's the reason for the change?